server/datastore/mysql/schema.sql

```diff
@@ -343,6 +343,7 @@ CREATE TABLE `hosts` (
   UNIQUE KEY `idx_host_unique_orbitnodekey` (`orbit_node_key`),
   KEY `fk_hosts_team_id` (`team_id`),
   KEY `hosts_platform_idx` (`platform`),
+  KEY `idx_hosts_hardware_serial` (`hardware_serial`),
```

Comment:
we have discussed this in the proposal document, in the platform weekly and in Slack, at the end the common agreement was to not enforce uniqueness of serial numbers because in non-macOS platforms it's acceptable to have more than one host row with the same serial.

I'd say let's move forward with this at least for now